CHANGELOG.txt

```diff
@@ -2,8 +2,16 @@
 
   Users side :
   ------------
+    # Fix XSS issue in popover title (backport 2.1)
+    # Fix dialog title of entity filter details using "<" (backport 2.1)
```

Comment:
In 2.0 there is no dialog info for filters

Reply:
Indeed :smile: